spec/features/admin/facilities/index_spec.rb

```diff
@@ -73,4 +74,15 @@
 
     expect(page).to have_content(unassociated_facility02.name)
   end
+
+  it "verify manage section overlay for the given permission" do
+    navigation.select_main_menu_tab("Manage")
```

Comment:
Looks like we have a mix of single and double quotes. Can we use single-quotes consistently throughout this PR?